src/try.js

```diff
@@ -0,0 +1,62 @@
+export {
+    typesOfValues,
+    tsInterfaceOutput
+}
+
+var x =
+{
+    "id": 0,
+    "name": "Blanche Mathews",
+    "age": "20",
+    "isSmiling": true
+}
+
+
+// var x = [
+//     {
+//         "id": 0,
+//         "name": "Blanche Mathews",
+//         "age":"20",
+//         "isSmiling": true
+//     },
+//     {
+//         "id": 1,
+//         "name": "Salinas Bruce",
+//         "isSmiling": false
+//     },
+//     {
+//         "id": 2,
+//         "name": "Bowman Santana",
+//         "isSmiling": true
+//     }
+// ]
+
+
+// console.log(...x);
+
+// var y = JSON.stringify(...x,)
+
+// var y = JSON.stringify(x).split(",")
+
+
+function typesOfValues(x) {
```

Comment:
whats the point of this function?
how is it any good, if I can't map the values back to the keys?

Reply:
function was written for tests. I'll leave it here for now ❤